client/app.jsx

```diff
@@ -70,6 +41,11 @@ export default class App extends React.Component {
   }
 
   resetSearch() {
+    const navLinks = document.getElementsByClassName('nav-link');
+    for (let i = 0; i < navLinks.length; i++) {
+      navLinks[i].className = 'nav-link';
+    }
```

Comment:
What exactly are you trying to do here? This is not ok in a React app. We generally do not reach into the DOM and start messing with stuff.